Source/Core/Core/NetPlayClient.cpp

```diff
@@ -632,6 +636,52 @@ void NetPlayClient::SendAsync(sf::Packet&& packet)
 // called from ---NETPLAY--- thread
 void NetPlayClient::ThreadFunc()
 {
+  bool qos_success = false;
+#if defined(_WIN32)
+  QOS_VERSION ver = {1, 0};
+
+  if (SConfig::GetInstance().bQoSEnabled && QOSCreateHandle(&ver, &m_qos_handle))
+  {
+    struct sockaddr_in sin = {0};
+
+    sin.sin_family = AF_INET;
+    sin.sin_port = ENET_HOST_TO_NET_16(m_server->host->address.port);
+    sin.sin_addr.s_addr = m_server->host->address.host;
+
+    if (QOSAddSocketToFlow(m_qos_handle, m_server->host->socket, reinterpret_cast<PSOCKADDR>(&sin),
+                           QOSTrafficTypeControl, QOS_NON_ADAPTIVE_FLOW, &m_qos_flow_id))
+    {
+      DWORD dscp = 0x2e;
+
+      // Sets DSCP to the same as Linux (0x2e)
+      // This will fail if we're not admin, but we ignore it
+      QOSSetFlow(m_qos_handle, m_qos_flow_id, QOSSetOutgoingDSCPValue, sizeof(DWORD), &dscp, 0,
+                 nullptr);
+
+      qos_success = true;
+    }
+  }
+#else
+  if (SConfig::GetInstance().bQoSEnabled)
+  {
+#if defined(__linux__)
+    int priority = 7;
+    setsockopt(m_server->host->socket, SOL_SOCKET, SO_PRIORITY, &priority, sizeof(priority));
+#endif
+    int tos_val = 0xb8;
```

Comment:
What is this value?

Reply:
`priority` is the priority of the packet in the kernel.
`tos_val` is the ToS (https://en.wikipedia.org/wiki/Type_of_service) value to set on each packet (@shortfuse should know more, assuming that's the right ShortFuse)

Reply:
Maybe define the individual values as `constexpr` to make it more explicit? ie. if I'm not wrong, it should be something like:
```c++
constexpr int ecn = 0b10; // ECN Capable Transport, ECT(0)
constexpr int dcsp = 0b111000; // IP Precendence 7 (Network Control)
int tos_val = ecn << 6 | dcsp;
```
Might be wrong interpreting the spec and values tho.